src/v/security/acl.h

```diff
@@ -239,12 +256,23 @@ class acl_host {
         }
     }
 
+    friend std::ostream& operator<<(std::ostream&, const acl_host&);
+
 private:
     acl_host() = default;
 
     std::optional<ss::net::inet_address> _addr;
 };
 
+inline std::ostream& operator<<(std::ostream& os, const acl_host& host) {
+    if (host._addr) {
+        fmt::print(os, "{}", *host._addr);
+    } else {
+        os << "*";
```

Comment:
is this asterik strategic or should we print -`any_host` or smth more informative.

Reply:
we can print whatever on our side. when we return strings to kafka it expects `*` as the wildcard.

Reply:
got it. this comment would be helpful.

Reply:
any_host is nice. added that, too.